test/rest.test.js

```diff
@@ -1184,6 +1184,27 @@ describe('strong-remoting-rest', function() {
           .expect(500)
           .end(expectErrorResponseContaining({message: 'an error'}, done));
       });
+      
+      it('should return 500 for unhandled errors thrown from before hooks', function(done) {
+        var method = givenSharedStaticMethod(
+          function greet(cb) {
+            cb();
+          }
+        );
```

Comment:
Nice, this his much simpler and better.

One more improvement to make: when your shared method is a no-op like in this case, you don't need to provide any function to `givenSharedStaticMethod`, it will build one by default - see [test/rest.test.js#L1975](https://github.com/PradnyaBaviskar/strong-remoting/blob/fd721cfac710f23ff9d88854cba66fa3b08e5764/test/rest.test.js#L1975).